### DISCUSSION TITLE
How to change the predefined sizes in the UI?

### DISCUSSION THREAD

**Discussion by marinkitagawauwu** [lllyasviel/stable-diffusion-webui-forge#1040] [Q&A]
I updated to the newest Forge and can't seem to figure out how to change the predefined size for SD, XL, etc., it's not in the UI config and I don't see it in the settings either. This is for me really bothersome as I restart the UI often and have to always manually input different sizes. Is there a way to change it to my size and save it that way? Before it was possible through UI config file.

**Comment by pflky** [comment]
main_entry.py
